vcd/resource_vcd_org.go

```diff
@@ -297,20 +316,21 @@ func resourceOrgDelete(d *schema.ResourceData, m interface{}) error {
 	err = adminOrg.Delete(deleteForce, deleteRecursive)
 	if err != nil {
 		log.Printf("[DEBUG] Error deleting org %s: %s", orgName, err)
-		return err
+		return diag.FromErr(err)
 	}
 	log.Printf("[TRACE] Org %s deleted", orgName)
-	return nil
+	return diags
```

Comment:
Is there any reason we have defined `var diags diag.Diagnostics` and then use it in return (but nowhere else). Couldn't we simply return `nil`?

Reply:
I think the reason why they do it is that they utilize this new pattern of `diag` (which can accumulate errors and warnings) instead of `error`, but until we make use of it, the `return diags` (which is always `nil` so far) instead of just `nil` looks to be a bit more confusing (requiring extra scan)

Reply:
I agree with you @Didainius . Let's keep code consistent as we are not using it. I'll revert this to the previous pattern!